pkg/cmd/pipelineresource/create.go

```diff
@@ -20,20 +20,25 @@ import (
 	"os"
 	"sort"
 
+	"github.com/ghodss/yaml"
```

Comment:
needs to be sorted

Reply:
Sorry, can you clarify? What needs to be sorted?

Reply:
```
import (
	"errors"
	"fmt"
	"os"
	"sort"

	"github.com/AlecAivazis/survey/v2"
	"github.com/AlecAivazis/survey/v2/terminal"
	"github.com/ghodss/yaml"
	"github.com/spf13/cobra"
	"github.com/tektoncd/cli/pkg/cli"
	"github.com/tektoncd/cli/pkg/helper/file"
	validateinput "github.com/tektoncd/cli/pkg/helper/validate"
	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
)
```

It should be like this

Reply:
Good catch. Not sure why gofmt didn't handle this. Pushed a change with them reordered.

Reply:
:+1: